lib/Controller/PageController.php

```diff
@@ -54,15 +55,15 @@ public function __construct(string $appName,
 								IConfig $config,
 								IInitialStateService $initialStateService,
 								IFactory $languageFactory,
-								SocialApiController $socialApi) {
+								IManager $manager) {
 		parent::__construct($appName, $request);
 
 		$this->appName = $appName;
 		$this->config = $config;
 		$this->initialStateService = $initialStateService;
 		$this->languageFactory = $languageFactory;
 
-		$this->socialApi = $socialApi;
+		$this->socialApi = new SocialApiController($appName, $request, $manager, $config, $languageFactory);
```

Comment:
You shouldn't have to do this, any reason you removed the automated dependency injection?

Reply:
No, will put it back in a minute.

Reply:
Now I am getting an error message: "Could not resolve OCA\Contacts\Controller\PageController! Class OCA\Contacts\Controller\SocialApiController does not exist". Could you check?

Reply:
Seems to be a setup issue in my dev-env.